### ISSUE TITLE
Duplicate error when activating stage users

### ISSUE THREAD

**Issue by abompard** [fedora-infra/freeipa-fas#93]
I can't seem to be able to activate stage users anymore. This is on a fresh installation:
```
$ ipa stageuser-add dummy --first Dummy --last User
------------------------
Added stage user "dummy"
------------------------
  User login: dummy
  First name: Dummy
  Last name: User
  Full name: Dummy User
  Display name: Dummy User
  Initials: DU
  Home directory: /home/dummy
  GECOS: Dummy User
  Login shell: /bin/sh
  Principal name: dummy@EXAMPLE.COM
  Principal alias: dummy@EXAMPLE.COM
  Email address: dummy@example.com
  UID: -1
  GID: -1
  Password: False
  Kerberos keys available: False

$ ipa stageuser-activate dummy
ipa: ERROR: This entry already exists
```
Can you reproduce this @tiran ? It's probably linked to 9ded969f.

**Comment by tiran** [comment]
Sigh, ``stageuser_activate`` does not work as I thought. It creates a new user entry first, then deletes the stage entry. This causes a conflict.

I'll talk to my co-workers tomorrow.

**Comment by abompard** [comment]
Fixed by #95
